src/components/NewLexemeForm.vue

```diff
@@ -88,4 +100,13 @@ export default {
 	border-radius: $border-radius-base;
 	border-color: $border-color-base-subtle;
 }
+
+.wbl-snl-small-text {
+	@include small-text;
+}
+
+.wbl-snl-copyright {
+	font-style: italic;
+}
```

Comment:
I assume we could also merge this, and use just the one class:

```suggestion
.wbl-snl-copyright {
	@include small-text;
	font-style: italic;
}
```

To me that would seem slightly nicer, but the two classes are also okay. WDYT?

Reply:
There's no reason why it shouldn't all be in wbl-snl-copyright, but Sarai suggested a small-text class for the mixin so I followed that.

Reply:
Hey. I suggested creating separate text style classes so a style can be applied to multiple elements without having to include the same mixin every time. This won't be the case for  the `small-text` style, but `body` will for sure be reused. I thought this approach sounded more efficient, but it's just an inexperienced assumption.

Reply:
I think your suggestion is definitely efficient for a bigger component with more text, this one only has the license text in it though, if I'm not mistaking.

Reply:
We'll have to apply quite a few styles! Not all the styles will be used repeatedly, but still it would be better to take a unified approach.

<img width="823" alt="Screenshot 2022-02-25 at 10 34 30" src="https://user-images.githubusercontent.com/3887458/155691415-4b0cca86-be49-491c-bc23-b5f5fd5400b4.png">
s.

Reply:
thanks @sai-san!